plugins/katello/nightly/advanced/http_proxies.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: plugins/katello/documentation
+title: Http Proxies
+version: 'nightly'
+---
+
+# Http Proxies
+
+## Http Proxy Support
+
+Katello enables external http proxies (provided by utilities such as [squid](http://www.squid-cache.org/) for repository operations such as synchronization.
+
+Http proxies can be created and then assigned to product though bulk selection as well as for each individual repository. Additionally, Katello provides http proxy policies for products or repositories. Policies include:
+  * Using the global http proxy (the default)
+  * Using a specified http proxy other than the global http proxy
+  * Do not use any http proxy
+
+## Creation
+
+There are 2 ways of creating an Http Proxy for use in katello - through a rake task or though the Foreman UI.
+
+### Creating an Http Proxy with Rake
+
+bundle exec rake katello:update_default_http_proxy -- --name "A Great Http Proxy" --url "http://192.168.121.1:8888" --user admin --password redhat
```

Comment:
In production we prefer `foreman-rake` instead of `bundle exec rake` since there's no bundler in RPM-based installs. It also ensures the correct Ruby is used. It's also good practice to use [RFC 5737](https://tools.ietf.org/html/rfc5737) (IPv4 Address Blocks Reserved for Documentation).
```suggestion
foreman-rake katello:update_default_http_proxy -- --name "A Great Http Proxy" --url "http://192.0.2.1:8888" --user admin --password redhat
```

Reply:
I agree about the capitlization. I'm not sure which way if the preferred approach, but I'll go with whatever one is best.

Reply:
Also thank you for the RFC tip!

Reply:
Not a native speaker so I can't comment on whether it should be Http Proxy, HTTP Proxy or http proxy. @mcorr might have a more useful suggestion :)